### ISSUE TITLE
Centralize community pages datalists

### ISSUE THREAD

**Issue by simonsan** [collinstommy/aoe-library#2]
Hey there, I really like the page you're doing, I had the same idea a while ago as well so I collected some pages here already:
https://github.com/SiegeEngineers/aoc-reference-data/blob/master/data/community_pages.yaml

I chose `yaml` as a format, because it's easier to deal with in other tools (like e.g. bots, overlays) than straight csv (imho). I wonder what you think about to centralize the content in that yaml and let it be pulled in on your page. It would have the advantage that by using `aoc-reference-data` as a well established community repository it's easier to find for future contributors on Github.

If you don't know the SiegeEngineers already, it's a development collective with their own discord, you can join using this link and we could discuss further.
https://discordapp.com/invite/njAsNuD

**Comment by collinstommy** [comment]
Hey @simonsan that makes sense to me. I will contact you on discord.

**Comment by simonsan** [comment]
@collinstommy Hey there, how are changing to the data being treated in this moment. I would like to add a category to the data. Should i do that in the `community_pages.yaml` or how far is the merging progress?

Cheers ;)

**Comment by collinstommy** [comment]
yep, go ahead and add that to `community_pages.yaml`. I will manually add it to aoelibrary.com

I'm not pulling from this yet. Haven't had the time to do so. I would like to pull the some of the player data as well but I need to write a layer on top as I don't necessarily want to show all players.
